README.md
emacs-config
============

My configuration for emacs editor


.emacs.d/history
;; -*- mode: emacs-lisp; coding: utf-8-unix -*-
;; Minibuffer history file, automatically generated by `savehist'.

(setq savehist-minibuffer-history-variables '(TeX-macro-history ido-buffer-history search-ring buffer-name-history query-replace-history coding-system-history file-name-history minibuffer-history TeX-command-history ido-file-history))
(setq ido-buffer-history '("slam.tex"))
(setq search-ring '("mult" "without " "the fact" "changes" "appendix" "addcon" "\\end{p" "appendices" "\\newc" "\\" "Claim" "line" "figure" "long" "The pro" "Partil"))
(setq buffer-name-history '("slam.tex" "*shell*" "Sparc_Manual.tex"))
(setq query-replace-history '(":load bn.hs" "" "^M"))
(setq coding-system-history '("unix"))
(setq file-name-history '("/home/iensen/Dropbox/plog/explanatiuon/desc.tex" "/home/iensen/Dropbox/plog/explanatiuon/bt2.txt" "/home/iensen/Dropbox/classes/DP/final_report/functional_testing.tex" "/home/iensen/Dropbox/classes/DP/slidez/auto/preentation.el" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/final_report/PLOGPLAN.tex" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_pr/p2_3x3_new2.plog" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/plogrunner.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/main.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/possibleworld.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/main.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/plogrunner.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/main.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/possibleworld.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/main.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/.#planner.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/main.py" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_pr/out.txt" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_pr/robot_move.plog" "/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_pr/out.txt" "/home/iensen/spec/spec.tex" "/home/iensen/Dropbox/classes/DP/10.04/presentation.tex" "/home/iensen/Dropbox/classes/causality/bn/bn.hs" "/home/iensen/sparc_project/User_Manual/Sparc_Manual.tex" "/home/iensen/elps/Manual/Elps_Manual.tex" "/home/iensen/elps/Manual/Elps_Manual.log" "/home/iensen/elps/Manual/Elps_Manual.tex" "/home/iensen/sparc_project/User_Manual/Sparc_Manual.tex" "/home/iensen/elps/tests/ex61" "/home/iensen/elps/tests/ex60" "/home/iensen/elps/tests/ex59"))
(setq minibuffer-history '("shell" "prolog-mode" "shell" "haskell-mode" "TeX-evince-sync-view" "shell" "TeX-evince-sync-view" "shell" "kill-buffer" "shell" "kill-buffer" "shell" "sh" "kill-buffer" "shell" "replace-string" "TeX-evince-sync-view" "replace-string" "TeX-evince-sync-view" "prolog-mode"))
(setq TeX-command-history '("LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX" "LaTeX" "BibTeX"))
(setq ido-file-history '("desc.tex" "bt2.txt" "functional_testing.tex" "preentation.el" "PLOGPLAN.tex" "p2_3x3_new2.plog" "plogrunner.py" "main.py" "possibleworld.py" "main.py" "plogrunner.py" "main.py" "possibleworld.py" "main.py" ".#planner.py" "main.py" "spec.tex" "presentation.tex" nil "bn.hs" "Sparc_Manual.tex" "Elps_Manual.tex" "Elps_Manual.log" "Elps_Manual.tex" "Sparc_Manual.tex" "ex17" "ex61" "ex60" "ex59" "ex58"))


.emacs.d/predictive/latex/dict-latex-preamble.el
(eval-when-compile (require 'cl))
(require 'dict-tree)
(defvar dict-latex-preamble nil "Dictionary dict-latex-preamble.")
(setq dict-latex-preamble '[cl-struct-dictree- "dict-latex-preamble" "/home/iensen/.emacs.d/predictive/latex/dict-latex-preamble" t nil < + #[514 "\301\300\302!\302!\"\"\210\207" [+ dictree--cell-set-data dictree--cell-data] 8 "

(fn NEW OLD)"] predictive-dict-rank-function #[514 "\300@\301A!B@\301A!B\"\207" [predictive-dict-rank-function dictree--cell-data] 7 "

(fn A B)"] time synchronize nil nil nil nil #s(hash-table size 65 test equal rehash-size 1.5 rehash-threshold 0.8 data ()) 0.1 nil nil #s(hash-table size 65 test equal rehash-size 1.5 rehash-threshold 0.8 data ()) 0.1 nil nil nil nil nil nil [cl-struct-trie- [nil [cl-struct-avl-tree- [[[nil nil [92 [cl-struct-avl-tree- [[[[nil nil [68 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [99 [cl-struct-avl-tree- [[nil nil [108 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [77 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [79 [cl-struct-avl-tree- [[nil nil [112 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil [nil nil [42 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] [--trie--terminator (0)] 1] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil nil [102 [cl-struct-avl-tree- [[nil [nil nil [117 [cl-struct-avl-tree- [[nil nil [115 [cl-struct-avl-tree- [[nil nil [115 [cl-struct-avl-tree- [[nil nil [121 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [111 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[[nil nil [101 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [99 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [100 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil nil [115 [cl-struct-avl-tree- [[[nil nil [101 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [115 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil nil [105 [cl-struct-avl-tree- [[nil nil [122 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [104 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [112 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [102 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [109 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [108 [cl-struct-avl-tree- [[nil nil [121 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 1] nil nil 0] nil]] 0] [98 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [123 [cl-struct-avl-tree- [[nil nil [100 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [99 [cl-struct-avl-tree- [[nil nil [117 [cl-struct-avl-tree- [[nil nil [109 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [125 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [[[nil nil [109 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [107 [cl-struct-avl-tree- [[[nil nil [98 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil nil [114 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [108 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [102 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil nil [112 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil [nil nil [115 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [121 [cl-struct-avl-tree- [[nil nil [108 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [110 [cl-struct-avl-tree- [[nil nil [117 [cl-struct-avl-tree- [[nil nil [109 [cl-struct-avl-tree- [[nil nil [98 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 1] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [110 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [119 [cl-struct-avl-tree- [[[[nil nil [99 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil [nil nil [117 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [109 [cl-struct-avl-tree- [[nil nil [109 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [100 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 1] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil nil [102 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [101 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [118 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [109 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil [nil nil [116 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [109 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [115 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [118 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [98 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [120 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 1] [108 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil [nil nil [117 [cl-struct-avl-tree- [[nil nil [115 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [112 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [99 [cl-struct-avl-tree- [[nil nil [107 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [115 [cl-struct-avl-tree- [[[nil nil [101 [cl-struct-avl-tree- [[nil [nil nil [116 [cl-struct-avl-tree- [[[nil nil [99 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [117 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil nil [116 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[[nil nil [100 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [112 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [nil nil [119 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [100 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [104 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [105 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [108 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [103 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [108 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [99 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [102 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 1] nil nil 0] nil]] 0] [nil nil [116 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [112 [cl-struct-avl-tree- [[nil nil [99 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [117 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [108 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [112 [cl-struct-avl-tree- [[nil nil [112 [cl-struct-avl-tree- [[nil nil [121 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 1] [114 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [119 [cl-struct-avl-tree- [[nil nil [99 [cl-struct-avl-tree- [[nil nil [111 [cl-struct-avl-tree- [[nil nil [109 [cl-struct-avl-tree- [[nil nil [109 [cl-struct-avl-tree- [[nil nil [97 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [100 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [105 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil [nil nil [112 [cl-struct-avl-tree- [[nil nil [117 [cl-struct-avl-tree- [[nil nil [116 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [99 [cl-struct-avl-tree- [[nil nil [108 [cl-struct-avl-tree- [[nil nil [117 [cl-struct-avl-tree- [[nil nil [100 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil [nil nil [111 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [108 [cl-struct-avl-tree- [[nil nil [121 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [--trie--terminator (0)] 1] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 1] nil nil 0] nil]] 0] nil nil 0] nil]] 1] nil nil 0] nil]] 0] [nil nil [117 [cl-struct-avl-tree- [[nil nil [114 [cl-struct-avl-tree- [[nil nil [108 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] [104 [cl-struct-avl-tree- [[nil nil [121 [cl-struct-avl-tree- [[nil nil [112 [cl-struct-avl-tree- [[nil nil [104 [cl-struct-avl-tree- [[nil nil [101 [cl-struct-avl-tree- [[nil nil [110 [cl-struct-avl-tree- [[nil nil [--trie--terminator (0)] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] nil]] 0] nil nil 0] #[514 "\302!\262\302!\262	=\203 \211	=?\205% \303\207\211	=\203! \304\207\300\"\207" [< trie--terminator trie--node-split t nil] 5 "

(fn A B)"]]] < #[514 "\302!\262\302!\262	=\203 \211	=?\205% \303\207\211	=\203! \304\207\300\"\207" [< trie--terminator trie--node-split t nil] 5 "

(fn A B)"] #[514 "\300\301\300\302\211\211\303$#\207" [vector cl-struct-avl-tree- nil 0] 10 "

(fn CMPFUN SEQ)"] avl-tree-enter avl-tree-delete avl-tree-member avl-tree-mapc avl-tree-empty avl-tree-stack avl-tree-stack-pop avl-tree-stack-empty-p trie--avl-transform-for-print trie--avl-transform-from-read t] nil])
(trie-transform-from-read (dictree--trie dict-latex-preamble))
(unless (memq dict-latex-preamble dictree-loaded-list)
  (push dict-latex-preamble dictree-loaded-list))


.emacs.d/saveplace.el
;;; -*- coding: utf-8 -*-
(("/home/iensen/.emacs" . 1874)
 ("/home/iensen/.emacs.d/.recentf" . 1666)
 ("/home/iensen/.ido.last" . 26796)
 ("/home/iensen/ASPECT/ASPPROGS/comparer.py" . 262)
 ("/home/iensen/ASPECT/ASPPROGS/facts_CanonG3.txt" . 549)
 ("/home/iensen/Dropbox/NASA/report/_region_.tex" . 1171)
 ("/home/iensen/Dropbox/NASA/report/report.log" . 2179)
 ("/home/iensen/Dropbox/NASA/report/report_cor.tex" . 40)
 ("/home/iensen/Dropbox/NASA/scau_proof/proof.log" . 7962)
 ("/home/iensen/Dropbox/NASA/scau_proof/proof.tex" . 5491)
 ("/home/iensen/Dropbox/classes/DP/10.04/presentation.log" . 40055)
 ("/home/iensen/Dropbox/classes/DP/final_report/_region_.tex" . 1537)
 ("/home/iensen/Dropbox/classes/DP/final_report/functional_testing.aux" . 758)
 ("/home/iensen/Dropbox/classes/DP/final_report/functional_testing.log" . 19387)
 ("/home/iensen/Dropbox/classes/DP/final_report/functional_testing.tex" . 27133)
 ("/home/iensen/Dropbox/classes/DP/slidez/preentation.log" . 35042)
 ("/home/iensen/Dropbox/classes/DP/slidez/preentation.tex" . 10475)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/Report_2/slam.aux" . 945)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/Report_2/slam.log" . 11391)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/Report_2/slam.tex" . 22907)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/assgn2/Report/tex_sources/slam.tex" . 10664)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/final_report/PLOGPLAN.log" . 29238)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/final_report/PLOGPLAN.tex" . 48187)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/final_report/_region_.tex" . 13705)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/main.py" . 1973)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/planner.py" . 1135)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/plogrunner.py" . 2647)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_planner/possibleworld.py" . 3182)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_pr/out.txt" . 1777)
 ("/home/iensen/Dropbox/classes/ProbabilisticRobotics/plog_pr/p2_3x3_new2.plog" . 2974)
 ("/home/iensen/Dropbox/classes/causality/bn/bn2.hs" . 8784)
 ("/home/iensen/Dropbox/classes/causality/bn/doc/doc.log" . 4973)
 ("/home/iensen/Dropbox/classes/causality/bn/doc/doc.tex" . 2872)
 ("/home/iensen/Dropbox/classes/causality/bn/spec/spec.tex" . 754)
 ("/home/iensen/Dropbox/olymp_problems/ttu_training/03.25.2014/2012-B/gen.py" . 541)
 ("/home/iensen/Dropbox/olymp_problems/ttu_training/04.08.2014/tutorial/overview.log" . 39879)
 ("/home/iensen/Dropbox/olymp_problems/ttu_training/04.08.2014/tutorial/overview.tex" . 21321)
 ("/home/iensen/Dropbox/olymp_problems/ttu_training/CookBook/cookbook.log" . 40434)
 ("/home/iensen/Dropbox/olymp_problems/ttu_training/CookBook/cookbook.tex" . 10558)
 ("/home/iensen/Dropbox/plog/correction/claim.tex" . 735)
 ("/home/iensen/Dropbox/plog/explanatiuon/desc.log" . 4457)
 ("/home/iensen/Dropbox/plog/explanatiuon/desc.tex" . 397)
 ("/home/iensen/draft/ELPS_20140408/ELPS.log" . 20417)
 ("/home/iensen/draft/ELPS_20140408/ELPS.tex" . 2081)
 ("/home/iensen/draft/assgn2/Report/tex_sources/slam.log" . 11250)
 ("/home/iensen/draft/assgn2/Report/tex_sources/slam.tex" . 12644)
 ("/home/iensen/elps/Manual/Elps_Manual.log" . 15514)
 ("/home/iensen/elps/Manual/Elps_Manual.tex" . 3484)
 ("/home/iensen/plog_pr/robot_move.plog" . 282)
 ("/home/iensen/sparc_project/User_Manual/Sparc_Manual.log" . 17620)
 ("/home/iensen/sparc_project/User_Manual/Sparc_Manual.tex" . 744)
 ("/home/iensen/spec/spec.log" . 9276)
 ("/home/iensen/spec/spec.tex" . 1465))
